### PR TITLE
South Korea update

### PR DESCRIPTION
**Pull request by MarcinMisztalTR**
Hello,
Purpose of this PR is to add part of the missing electoral districts:
https://en.wikipedia.org/wiki/South_Korean_Legislature_Constituencies

### CODE REVIEW

File: identifiers/country-kr.csv [hunk 1]
```diff
@@ -17,6 +17,13 @@ ocd-division/country:kr/ed:bucheon_city_1,Bucheon City First
 ocd-division/country:kr/ed:bucheon_city_2,Bucheon City Second
 ocd-division/country:kr/ed:bucheon_city_3,Bucheon City Third
 ocd-division/country:kr/ed:bucheon_city_fourth,Bucheon City Fourth
+ocd-division/country:kr/ed:buk_district,Buk District
+ocd-division/country:kr/ed:buk_district_and_gangseo_district_1,Buk District and Gangseo District First
+ocd-division/country:kr/ed:buk_district_and_gangseo_district_2,Buk District and Gangseo District Second
+ocd-division/country:kr/ed:bukd_district_1,Buk District First Daegu
```
Comment (HKSenior) [inline identifiers/country-kr.csv:23]: Can we make these ids a bit more descriptive by including the full city name? Keeping it consistent with other ids such as `ocd-division/country:kr/ed:anyang_city_dongan_district_1`.

```suggestion
ocd-division/country:kr/ed:daegu_city_buk_district_1,Buk District First Daegu
```

File: identifiers/country-kr/single_member_constituencies.csv [hunk 2]
```diff
@@ -251,4 +251,32 @@ ocd-division/country:kr/ed:yongin_city_3,Yongin City Third
 ocd-division/country:kr/ed:yongin_city_four,Yongin City Four
 ocd-division/country:kr/ed:yongsan,Yongsan
 ocd-division/country:kr/ed:yuseong_district_1,Yuseong District First
-ocd-division/country:kr/ed:yuseong_district_2,Yuseong District Second
\ No newline at end of file
+ocd-division/country:kr/ed:yuseong_district_2,Yuseong District Second
+ocd-division/country:kr/ed:buk_district,Buk District
+ocd-division/country:kr/ed:buk_district_and_gangseo_district_1,Buk District and Gangseo District First
+ocd-division/country:kr/ed:buk_district_and_gangseo_district_2,Buk District and Gangseo District Second
+ocd-division/country:kr/ed:bukd_district_1,Buk District First Daegu
```
Comment (HKSenior) [inline identifiers/country-kr/single_member_constituencies.csv:258]: Same here and below.

```suggestion
ocd-division/country:kr/ed:daegu_city_buk_district_1,Buk District First Daegu
```

**Review by HKSenior — CHANGES_REQUESTED** [review]

**Comment by MarcinMisztalTR** [comment]
Hello, Thank you for the review. I've updated part of the districts which contained shortcuts in the ocdid value.

File: identifiers/country-kr/single_member_constituencies.csv [hunk 3]
```diff
@@ -251,4 +251,32 @@ ocd-division/country:kr/ed:yongin_city_3,Yongin City Third
 ocd-division/country:kr/ed:yongin_city_four,Yongin City Four
 ocd-division/country:kr/ed:yongsan,Yongsan
 ocd-division/country:kr/ed:yuseong_district_1,Yuseong District First
-ocd-division/country:kr/ed:yuseong_district_2,Yuseong District Second
\ No newline at end of file
+ocd-division/country:kr/ed:yuseong_district_2,Yuseong District Second
+ocd-division/country:kr/ed:buk_district,Buk District
+ocd-division/country:kr/ed:buk_district_and_gangseo_district_1,Buk District and Gangseo District First
```
Comment (jloutsenhizer) [inline identifiers/country-kr/single_member_constituencies.csv:256]: I think these new OCD IDs are duplicates. According to https://en.wikipedia.org/wiki/South_Korean_Legislature_Constituencies there should only be 253 districts. I took a look at a few of these and I think they already exist. Some examples (from new district => old district):

- `ocd-division/country:kr/ed:gwangju_city_buk_district_1` => `ocd-division/country:kr/ed:gwangju_buk_district_1`
- `ocd-division/country:kr/ed:buk_district_and_gangseo_district_1` => `ocd-division/country:kr/ed:busan_buk_district_and_gangseo_district_1`
- `ocd-division/country:kr/ed:seo_district` => `ocd-division/country:kr/ed:daegu_seo_district`
- `ocd-division/country:kr/ed:daejeon_city_jung_district` => `ocd-division/country:kr/ed:daejeon_jung_district`

Comment (MarcinMisztalTR) [reply identifiers/country-kr/single_member_constituencies.csv:256]: Thank you for your response and review. You are right, we have double checked our sources and all of the ocdids listed in this PR already exists. We'll correct it at our side. 

I believe this PR can be closed.

Comment (jloutsenhizer) [reply identifiers/country-kr/single_member_constituencies.csv:256]: Thanks for confirming, I'll close this

**Pull request closed without merging**